docs/partners/plan-provisioning-direct-api.md

```diff
@@ -14,19 +14,19 @@ A successful response will include a JSON object with several keys. We are speci
 
 For more detailed information about oAuth on WordPress.com, visit the [documentation on oAuth2 authentication](https://developer.wordpress.com/docs/oauth2/).
 
-### Endpoint Information (/oauth2/token)
```

Comment:
@gravityrail – I originally added these so that the header links were unique. This would facilitate sending a link to a specific section of the doc. It also got rid of some Markdown sniffer warnings.